raiden_contracts/contracts/SecretRegistry.sol

```diff
@@ -22,7 +22,7 @@ contract SecretRegistry {
     /// @param secret The secret used to lock the hash time lock.
     /// @return true if secret was registered, false if the secret was already registered.
     function registerSecret(bytes32 secret) public returns (bool) {
-        bytes32 secrethash = keccak256(secret);
+        bytes32 secrethash = keccak256(abi.encodePacked(secret));
```

Comment:
Hmm here also it's 1 single argument and it's also bytes

Reply:
secret is `bytes32`. It expects `bytes`. So yes, compiler gave an error here.